topologies/atedut_3.testbed

```diff
@@ -0,0 +1,40 @@
+duts {
```

Comment:
Tests that need 3 ports should just use the atedut_4.testbed. Delete this file?

Reply:
Yes, removed for now. We may have to revisit this later